### DISCUSSION TITLE
How to use snipets, containing `

### DISCUSSION THREAD

**Discussion by aisbergde** [gsantner/markor#1867] [Q&A] [answered]
Maybe a bug, but I am afraid to create bug reports after last feedback I got. So I ask this as a question.

It is very hard to find "`" on my keyboard. So I created code snippets, containing these characters. But when I use these snippets, then these characters do not appear. It looks like they are filtered out. Other code snippets containing other characters are working fine. 

What should or could I do?

It would be great, if some of the action items, which are available in Markdown files, would be available in text files. We have no specific type adoc, I have to use the text type, and actions are limited in text type.

**Comment by harshad1** [comment, marked as answer]
Escape them with a backslash (snippets use backticks to delimit dates) i.e. use `` \` `` to insert a literal `` ` ``

As for action items. Is there any reason you can't just change the format to markdown? You can set the ft independently of the extension and it will be remembered by markor....

**Reply by aisbergde** [reply]
Thank you for the instructions regarding \ - this works fine.

I need the extension adoc, because these are AsciiDoc files. And they need to be detected as adoc for example on the desktop.

It looks like I will need to fork Markor, lean Java and implement an additional format type :-)

**Reply by harshad1** [reply]
I think you misunderstand me. 3 dots -> FileSettings -> Format -> Markdown

**Reply by harshad1** [reply]
Leave the file as `.adoc` and just change the markor type to markdown

**Reply by aisbergde** [reply]
I already started to implement full AsciiDoc support yesterday. The Markor code is well structured and I progress well. I have some issues to understand some regex functions, but in General I am doing fine.
